tools/harness-simulation/posix/sniffer_sim/sniffer.py

```diff
@@ -49,6 +51,7 @@ class CaptureState(enum.Flag):
     NONE = 0
     THREAD = enum.auto()
     ETHERNET = enum.auto()
+    SYNC = enum.auto()
```

Comment:
May document the capture states? 

The original 3 states are more self-explanatory, but `SYNC` is less obvious.

Reply:
I intended to support two modes:
- sync: transfer the packets to Windows once they are captured on POSIX
- async: transfer the whole capture file after stopping the sniffer

But it seems unnecessary as the user can hardly notice the differences when using Harness. The only difference that user can notice is that IP addresses do not need to be input manually when using `sync` mode in the manual DUT mode, making `sync` is always no worse than `async`.

For that reason, I consider removing the `SYNC` bit here as such mode is always used.

Reply:
In `SimSniffer.py`, `TransferFile` is called right after calling `Start`.
I think the case that calling `Stop` before calling `TransferFile` could not happen. Because Harness always sleeps for a few seconds after starting the sniffer and before stopping the sniffer.